idd/Energy+.idd.in

```diff
@@ -49467,13 +49467,106 @@ Coil:Cooling:Water:DetailedGeometry,
+       \type object-list
+       \object-list ScheduleNames
+   A5, \field Cooling Coil Object Type
+       \type choice
+       \required-field
+       \key Coil:Cooling:Water
+       \key Coil:Cooling:Water:DetailedGeometry
+       \key CoilSystem:Cooling:Water:HeatExchangerAssisted
+   A6, \field Cooling Coil Name
+       \required-field
+       \type object-list
+       \object-list CoolingCoilsWater
+   A7, \field Dehumidification Control Type
+       \type choice
+       \key None
+       \key Multimode
+       \key CoolReheat
+       \default None
+       \note None = meet sensible load only
+       \note Multimode = activate water coil and meet sensible load.       
+       \note If no sensible load exists, and Run on Latent Load = Yes, and a latent
+       \note load exists, the coil will operate to meet the latent load.
+       \note If the latent load cannot be met the heat exchanger will be activated.
+       \note IF Run on Latent Load = No, the heat exchanger will always be active.
+       \note This control mode either switches the coil mode or allows the heat exchanger to 
+       \note be turned on and off based on the zone dehumidification requirements. Valid 
+       \note only with cooling coil type CoilSystem:Cooling:Water:HeatExchangerAssisted.   
+       \note CoolReheat = cool beyond the dry-bulb setpoint as required to meet the 
+       \note humidity setpoint. Valid with all cooling coil types. When a heat exchanger 
+       \note assisted cooling coil is used, the heat exchanger is locked on at all times.      
+       \note For all dehumidification controls, the max
+       \note humidity setpoint on the Sensor Node is used.
+       \note SetpointManager:SingleZone:Humidity:Maximum,
+       \note SetpointManager:MultiZone:Humidity:Maximum, or
+       \note SetpointManager:MultiZone:MaximumHumidity:Average, and
+       \note SetpointManager:OutdoorAirPretreat (optional) objects.
+   A8, \field Run on Sensible Load
+       \type choice
+       \key Yes
+       \key No
+       \default Yes
+       \note If Yes, unit will run if there is a sensible load.
+       \note If No, unit will not run if there is only a sensible load.
+       \note Dehumidification controls will be active if specified.
+   A9, \field Run on Latent Load
+       \type choice
+       \key Yes
+       \key No
+       \default No
+       \note If Yes, unit will run if there is a latent load.
+       \note even if there is no sensible load.
+       \note If No, unit will not run only if there is a latent load.
+       \note Dehumidification controls will be active if specified.
+   N1; \field Minimum Air To Water Temperature Offset
+       \note Coil will turn on as required when inlet air temperature is above
+       \note water temperature by amount of offset. To model a waterside
+       \note economizer connect to condenser loop and increase offset as desired.
+       \type real
+       \units C
+       \minimum 0.0
+       \default 0.0 
```

Comment:
Updated the IDD to correct key choices in *Dehumidification Control Type* input field.